torch/_meta_registrations.py

```diff
@@ -5126,8 +5126,8 @@ def meta__scaled_dot_product_flash(
     return (
         attention,
         logsumexp,
-        None,
-        None,
+        torch.empty((), dtype=torch.long, device="meta"),
+        torch.empty((), dtype=torch.long, device="meta"),
```

Comment:
Test failures are real, the ATen version returns undefined tensors which probably become None when returned to python.

Reply:
@peterbell10 Thanks for the info! Sounds like we should fix the schema in native_functions.yaml then?

Reply:
The schema isn't wrong per-se as it's normal for internal operators to return undefined tensors. For example in the backward pass all-zero tensors are commonly represented as undefined tensor.

Reply:
@peterbell10 Hmm, I guess it is valid but still surprising to users who saw `Tensor` as return type but observe None returns from `torch.ops.aten._scaled_dot_product_flash_attention`. 

Context: In export we're trying to serialize the meta['val'] field in an FX graph and we did a little bit of type checking to see whether meta['val'] matches with type in the op schema. Seem that there's an implicit conversion between Python `None` and torch undefined tensor here, so we will instead change how we do the type checking. A bit annoying fact to learn but seems okay to me. 

thanks for the explanation.